ui/src/app/views/id-verification-page/component.html

```diff
@@ -1,40 +0,0 @@
-<h2>Identity Verification</h2>
-<div>
-  <h3>Basic Information</h3>
-  <div>
-    <label for="first-name">First name</label>
-    <input id="first-name" type="text" size="30" autofocus [(ngModel)]="request.firstName">
-  </div>
-  <div>
-    <label for="last-name">Last name</label>
-    <input id="last-name" type="text" size="30" [(ngModel)]="request.lastName">
-  </div>
-  <div>
-    Address<br>
```

Comment:
So we're entirely getting rid of the old ID verification stuff? (Are we going to reintroduce something like this later when we integrate with a new vendor?)